### PR TITLE
Add endpoints to get collection IDs

### PR DESCRIPTION
**Pull request by MrCreosote**
No way to figure out what inactive collections are available otherwise

### CODE REVIEW

File: src/service/routes_collections.py [hunk 1]
```diff
@@ -192,6 +205,13 @@ async def save_collection(
     return sc
 
 
+@ROUTER_COLLECTIONS_ADMIN.get("/collectionids/all/", response_model = CollectionIDs)
```
Comment (MrCreosote) [inline src/service/routes_collections.py:208]: I'm not totally sold on the endpoint uris. If you have a better idea let me know

File: src/service/storage_arango.py [hunk 2]
```diff
@@ -27,23 +27,6 @@
 ARANGO_ERR_NAME_EXISTS = 1207
 _ARANGO_ERR_UNIQUE_CONSTRAINT = 1210
 
-_QUERY_GET_NEXT_VERSION = f"""
```
Comment (MrCreosote) [inline src/service/storage_arango.py:30]: Easier to read the methods if the queries are inline. If the same aql is used for multiple methods then a module global might make sense

File: src/service/storage_arango.py [hunk 3]
```diff
@@ -123,15 +107,22 @@ async def create(
 
     def __init__(self, db: StandardDatabase):
         self._db = db
-        # TODO DB make some sort of pluginish system for data products
```
Comment (MrCreosote) [inline src/service/storage_arango.py:126]: Duplicated by a TODO in app.py

**Comment by MrCreosote** [comment]
*)&*))8&(*^&^) codecov

**Review by Tianhao-Gu — APPROVED** [review]
👍

**Pull request merged**